app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,9 @@
 <resources>
     <string name="app_name">Trakt</string>
+    <string name="title_activity_main">MainActivity</string>
+
+    <string name="navigation_drawer_open">Open navigation drawer</string>
+    <string name="navigation_drawer_close">Close navigation drawer</string>
+
+    <string name="action_settings">Settings</string>
```

Comment:
Where is `action_settings` used? If it is in the drawer, it would be better to rename it to `navigation_drawer_settings`.